hack/get-root-key.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# /*
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+echo ""
+kubectl get secret vsecm-safe-age-key -n vsecm-system \
```

Comment:
This will be useful for the manual input mode.

I am planning to create a demo video for this use case too.